src/App.js

```diff
@@ -1,12 +0,0 @@
-import logo from './logo.svg';
```

Comment:
As react gives you the freedom of moving things around and structure them the way you want, it's recommended to keep the structure as is and build over it, especially the App.js. It's recommended to keep it as is in your root src directory